scripts/foundry/governance/RevokeMultiSig.s.sol

```diff
@@ -0,0 +1,266 @@
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Change Gauge Controller admin */
+            {
+                to = _chainToContract(chainId, ContractType.GaugeController);
+                bytes memory data = abi.encodeWithSelector(
+                    IGaugeController.commit_transfer_ownership.selector,
+                    address(_chainToContract(chainId, ContractType.Governor))
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Change Gauge Sushi ANGLE-agEUR admin */
+            {
+                to = gaugeSushiAngleAgEUR;
+                bytes memory data = abi.encodeWithSelector(
+                    ILiquidityGauge.commit_transfer_ownership.selector,
+                    address(_chainToContract(chainId, ContractType.Governor))
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Add the on chain governor as governor of Angle Distributor  */
+            {
+                to = _chainToContract(chainId, ContractType.AngleDistributor);
+                bytes memory data = abi.encodeWithSelector(
+                    IAccessControl.grantRole.selector,
+                    governorRole,
+                    address(_chainToContract(chainId, ContractType.Governor))
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Add the on chain governor as guardian of Angle Distributor  */
+            {
+                to = _chainToContract(chainId, ContractType.AngleDistributor);
+                bytes memory data = abi.encodeWithSelector(
+                    IAccessControl.grantRole.selector,
+                    guardianRole,
+                    address(_chainToContract(chainId, ContractType.Governor))
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            /** Remove the multisig as guardian of Angle Distributor  */
+            {
+                to = _chainToContract(chainId, ContractType.AngleDistributor);
```

Comment:
Let's keep the multisig as guardian no? So the emergency 4/6 multisig has all the same roles as the 2/3 msig